packages/insomnia-smoke-test/playwright.config.ts

```diff
@@ -19,5 +19,6 @@ const config: PlaywrightTestConfig = {
   forbidOnly: !!process.env.CI,
   outputDir: 'screenshots',
   testDir: 'tests',
+  workers: process.env.CI ? 1 : 4,
```

Comment:
We should use os.cpus().length for the number of workers.
Also not sure if adding more workers in dev is useful atm since we don't have many tests.
Perhaps we can change this to accept an arg/env for parallelism and if so use the number of the machine cores.
e.g. 
```suggestion
  workers: process.env.PW_CONCURRENT ? require('os').cpus().length : 1,
```